lib/measured/measurable.rb

```diff
@@ -1,6 +1,8 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /\A\s*([0-9.]+)\s*([a-zA-Z_-]+)\s*\Z/
```

Comment:
So the fact that we need a review comment means we should do something about the regex clarity. Free-spacing regex is pretty fantastic, by using `/x` which ignores whitespaces.

```ruby
# before
/\A\s*([0-9.]+)\s*([a-zA-Z_-]+)\s*\Z/

# after
/
  \A                # start of line
  \s*               # any number of blank characters
  ([0-9.]+)         # one or more digits or decimals
  \s*               # any number of blank characters
  ([a-zA-Z_-]+)     # one or more letters, dashes, or underscores
  \s*               # any number of blank characters
  \Z                # end of line
/x
```